content/en/smart-contracts/fundamentals/the-filecoin-virtual-machine/index.md

```diff
@@ -23,222 +23,44 @@ aliases:
     - "/developers/smart-contracts/about/use-cases/"
 ---
 
-## Features
+## Introduction
 
-The Filecoin virtual machine (FVM) was created to enable developers to build new use cases on top of the Filecoin network. Data access control, data DAOs, perpetual storage, collateral leasing, and Ethereum-compatible fungible and non-fungible tokens are just some of the use-cases for the FVM.
+Filecoin’s storage and retrieval capabilities can be thought of as the base layer of the Filecoin blockchain, and FVM can be thought of as a layer on top of Filecoin enabling programmatic execution of storage and retrieval.  
 
-### Data access control
+Whereas other blockchains do have smart contract capabilities, FVM’s smart contracts can use Filecoin storage and retrieval primitives with computational logic conditions.  FVM will also enable Layer 2 capabilities, such as “compute over data” and [content delivery networks](https://saturn.tech/).
 
-The FVM allows Filecoin network participants to limit individual access to certain data sets, an advantage previously only available using centralized storage solutions.
+Some additional notes about FVM’s technical specifications: 
 
-### Data DAO
+- WASM-based | The FVM is a WASM-based polyglot execution environment for IPLD data, meaning that FVM gives developers access to IPFS / IPLD data primitives and can accommodate smart contracts (actors) written in any programming language that compiles to WASM.  
 
-FVM data access control enables the creation and management of data centered decentralized-autonomous-organizations, which can govern and monetize data access and pool returns into a shared treasury.
+- FEVM Compatibility |  Are you an Ethereum / Solidity developer?  You can build the next killer app on FVM and make use of the [Filecoin Solidity library](https://docs.zondax.ch/fevm/filecoin-solidity/).  Learn more about how FVM is Ethereum runtime and solidity compatible in the next section. 
```

Comment:
Again, don't use the pipe `|` symbol. Linking to an external site right off the bat it a bad idea. Suggest linking to our internal Solidity page.

Reply:
Ok, we can make this change.